stream-chat-android-ui-uitests/src/androidTest/java/io/getstream/chat/android/uitests/snapshot/compose/ComposeScreenshotTest.kt

```diff
@@ -56,7 +60,15 @@ abstract class ComposeScreenshotTest : ScreenshotTest {
             ChatTheme(
                 isInDarkMode = isDarkMode,
                 imageLoaderFactory = { FakeImageLoader(context) },
-                content = content
+                content = {
```

Comment:
Most of the views in dark mode were white as they had transparent background and should inherit background from their parents. Adding black background to improve the way they are displayed in dark mode.